media_mpx.module

```diff
@@ -179,3 +180,11 @@ function media_mpx_migration_plugins_alter(array &$migrations) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function media_mpx_form_media_form_alter(&$form, $form_state, $form_id) {
+  $media_form_alter = MediaFormAlter::create(\Drupal::getContainer());
```

Comment:
Why create the object this way vs. declaring it as a service and then doing `\Drupal::service('media_form_service')`?

Reply:
Mainly to keep the services file "clean" of services that in reality do nothing out of this context, given they still need to be called from this `_alter()`. Having said that, I've no real preference here, just thought it wasn't adding any more value for a case like :).

Reply:
Fair enough. I like your rationale here, make a lot of sense. Especially as busy as the services file of this module already is!

Reply:
I think a more common pattern would be to call the constructor here. Is this class ever actually used by core in a way where it calls create(), or is this the only call?